kernel/kernel-api/src/main/java/io/delta/kernel/client/JsonHandler.java

```diff
@@ -52,6 +52,17 @@ public interface JsonHandler
      */
     ColumnarBatch parseJson(ColumnVector jsonStringVector, StructType outputSchema);
 
+    /**
+     * Parse the given {@code schemaString} according to the Delta Protocol
+     * <a href="https://github.com/delta-io/delta/blob/master/PROTOCOL.md#primitive-types">
+     *    serialization rules </a>.
+     *
+     * @param schemaString the JSON formatted schema string to parse
+     * @return the parsed {@link StructType}
```

Comment:
```suggestion
    /**
     * Deserialize the Delta table schema from {@code schemaString} according to the Delta Protocol
     * <a href="https://github.com/delta-io/delta/blob/master/PROTOCOL.md#primitive-types">
     *    schema serialization rules</a>.
     *
     * @param schemaString the JSON formatted schema string to parse
     * @return the parsed {@link StructType}
```

Reply:
@vkorukanti - if you want the text to say `Deserialize the Delta table schema ...` why don't we rename this method to `deserializeTableSchema`?